valohai_yaml/pipelines/conversion.py

```diff
@@ -7,29 +7,53 @@
     ExecutionNode,
     Node,
     Pipeline,
+    PipelineParameter,
     TaskNode,
 )
 from valohai_yaml.objs.pipelines.override import Override
 
 ConvertedObject = Dict[str, Any]
 
 
+if sys.version_info >= (3, 8):
+    from typing import TypedDict
+    class ConvertedPipeline(TypedDict):
```

Comment:
No change required from this commend.

`ConvertedPipeline` sounds like a pipeline that it self has been converted and not that the data presenting the same pipeline is converted into another form. I see that this follows the `ConvertedObject` naming convention but I would like to discuss if better name could be found, like `PipelineDict` or `PipelineData` or `SerializedPipeline`.

I had a short poll at the office and the end thought was that the `ConvertedPipeline` is the best option.